TrackingSystem/src/main/java/de/telran/entity/CustomerEntity.java

```diff
@@ -0,0 +1,23 @@
+package de.telran.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@Data
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+@AllArgsConstructor
+@NoArgsConstructor
+public class CustomerEntity {
```

Comment:
It failed here though since the table in the DB is called Customer, but the corresponding entity class is CustomerEntity, which is a mismatch you should fix either by annotating your class (check `@Table` annotation) or rename class to match the table name.